WooCommerce/Classes/ViewRelated/Survey/SurveyViewController.swift

```diff
@@ -140,6 +146,33 @@ extension SurveyViewController: WKNavigationDelegate {
         removeLoadingView()
     }
 }
+// MARK: Survey Tags
+//
+internal extension URL {
```

Comment:
nit: `internal` is the default and can be omitted :)

```suggestion
extension URL {
```

Reply:
Nice! I've submitted your suggestion, thanks for that information! 😄